### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by vior1992**

### CODE REVIEW

File: staff/daniel-villegas/app-template/about.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<body>
+    <header>
+        <nav>
+            <a href="./home.html">Home</a>
+            <a href="./login.html">Login</a>
+            <a href="./news.html">News</a>
+            <a href="./register.html">Register</a>
+            <a href="./about.html">About</a>
+            <a href="./contact.html">Contact</a>
+            <a href="./profile.html">Profile</a>
```
Comment (aleixcam) [inline staff/daniel-villegas/app-template/about.html]: the anchors inside a ```<nav>``` menu are better inside a ```<li>``` inside an unordered list (```<ul>```)

File: staff/daniel-villegas/app-template/contact.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+<body>
+    <header>
+        <nav>
+            <a href="./home.html">Home</a>
+            <a href="./login.html">Login</a>
+            <a href="./news.html">News</a>
+            <a href="./register.html">Register</a>
+            <a href="./about.html">About</a>
+            <a href="./contact.html">Contact</a>
+            <a href="./profile.html">Profile</a>
```
Comment (aleixcam) [inline staff/daniel-villegas/app-template/contact.html]: same as above

File: staff/daniel-villegas/app-template/contact.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    </header>
+
+
+
+    <section>
+        <p>Puedes contactarnos a traves de:</p>
+        <ul>
+            <li>Telefono: 982189373</li>
+            <li>Fax: 982189379</li>
+            <li><a href="falso@hotmail.com">Email</a></li>
```
Comment (aleixcam) [inline staff/daniel-villegas/app-template/contact.html]: for a mail anchor may you want to use ```href="mailto:example@mail.com"```

File: staff/daniel-villegas/app-template/home.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+<body>
+    <header>
+        <nav>
+                <a href="./home.html">Home</a>
+                <a href="./login.html">Login</a>
+                <a href="./news.html">News</a>
+                <a href="./register.html">Register</a>
+                <a href="./about.html">About</a>
+                <a href="./contact.html">Contact</a>
+                <a href="./profile.html">Profile</a>
```
Comment (aleixcam) [inline staff/daniel-villegas/app-template/home.html]: ditto

File: staff/daniel-villegas/app-template/about.html (hunk 1 above)
Comment (vior1992) [reply staff/daniel-villegas/app-template/about.html]: Indeed. Changed!

File: staff/daniel-villegas/app-template/contact.html (hunk 2 above)
Comment (vior1992) [reply staff/daniel-villegas/app-template/contact.html]: Indeed. Changed!

File: staff/daniel-villegas/app-template/home.html (hunk 4 above)
Comment (vior1992) [reply staff/daniel-villegas/app-template/home.html]: Indeed. Changed!

**Pull request merged**